package.json

```diff
@@ -103,18 +105,21 @@
     "svg-sprite-loader": "3.9.2",
     "svgstore-cli": "1.3.1",
     "travis-deploy-once": "5.0.10",
-    "url-loader": "^1.1.2",
+    "url-loader": "1.1.2",
     "webpack": "4.27.1"
   },
   "dependencies": {
     "@babel/runtime": "^7.3.4",
     "body-scroll-lock": "^2.5.8",
     "classnames": "^2.2.5",
+    "cozy-client": "6.26.0",
```

Comment:
Should not be dep, right? It should be a peerDep and devDep ?

Reply:
No I don't think so. [`cozy-client` is really used in `cozy-ui` now](https://github.com/cozy/cozy-ui/pull/962/files#diff-670d8a9e3dcb209b4f42fe9dd43ca070R3) so it's a real dependency.

Reply:
And? 

We use cozy-client in Harvest, but cozy-client is a peerdep. This is the responsibility of the App to require the lib. I though we already talked about our peerDep for our internal libs. Nop?

Reply:
>  I though we already talked about our peerDep for our internal libs.

Was there a consensus on that?

I have to re-read the docs every time, but I'm fairly sure perDependency is wrong in this case. The viewer is not just "hooking into" cozy-client, it's not a plugin for cozy-client. It literally needs cozy-client to run.

It's confusing because the app has to initiate cozy-client and insert it into the context. So in that sense, both the app and the viewer need cozy-client. But from what I read peerDependencies mean that only the host actually *uses* the dependency, which isn't the case here.